docs/operating-eck/configure-operator-metrics.asciidoc

```diff
@@ -14,7 +14,7 @@ The ECK operator provides a metrics endpoint that can be used to monitor the ope
 * <<{p}-securing-the-metrics-endpoint,Securing the metrics endpoint>>
 * <<{p}-prometheus-requirements,Prometheus requirements>>
 
-NOTE: The ECK operator metrics endpoint will be secured by default beginning in version 2.14.0.
+NOTE: The ECK operator metrics endpoint will be secured by default beginning in version 3.0.0.
```

Comment:
Not sure if we should include that. We did not follow up with the previous announment of making this the default in 2.14.